app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
```

Comment:
giải thích code này nghe bé ơi

Reply:
Trong đoạn code trên sử dụng toán tử gán ||= sẽ trả về giá trị của @current_user nếu giá trị là đúng hoặc gán kết quả của User.find_by id: session[:user_id] cho @current_user